oereb_einzelschutz_naturreservat/job.properties

```
@@ -0,0 +1,3 @@
+parameters.stringParam:buildDescription;Keine Beschreibung angegeben;Beschreibung/Grund f&uuml;r die Publikation der Daten
```

Ich habe die richtige Lösung für das "ISO-8859-1-Problem" gefunden, siehe #144. Nun kann man ganz normal "ü" verwenden, und das File darf/muss UTF-8 sein.
```suggestion
parameters.stringParam:buildDescription;Keine Beschreibung angegeben;Beschreibung/Grund für die Publikation der Daten
```

---

Ist angepasst. gretl-users war auch noch falsch.